### PR TITLE
Add more tests to logger package (#804 revamp)

### PR DESCRIPTION
**Pull request by seratch**
###  Summary

This pull request merges https://github.com/slackapi/node-slack-sdk/pull/804 into the latest `main` branch. 

@43081j 
So sorry for taking a long time to merge this.

### Requirements (place an `x` in each `[ ]`)

* [x] I've read and understood the [Contributing Guidelines](https://github.com/slackapi/node-slack-sdk/blob/master/.github/contributing.md) and have done my best effort to follow them.
* [x] I've read and agree to the [Code of Conduct](https://slackhq.github.io/code-of-conduct).

### CODE REVIEW

File: packages/logger/src/index.spec.js [hunk 1]
```diff
@@ -1,42 +1,147 @@
...
 const { ConsoleLogger, LogLevel } = require('./index');
 
-describe('logger', () => {
-  it('should have the default LogLevel', () => {
-    const logger = new ConsoleLogger();
-    assert.equal(logger.getLevel(), LogLevel.INFO);
-  });
+describe('Logger', () => {
+  describe('ConsoleLogger', () => {
+    let logger; // Logger
```
Comment (seratch) [inline packages/logger/src/index.spec.js:7]: I modified this part as the original `let logger: Logger;` does not work.

File: packages/logger/src/index.spec.js [hunk 2]
```diff
@@ -1,42 +1,147 @@
...
+        assert.deepEqual(infoStub.firstCall.args, [
+          '[INFO] ',
+          'foobles',
+          'test',
+        ]);
+      });
+    });
+
+    describe('log levels', () => {
+      let debugStub; // sinon.SinonStub
```
Comment (seratch) [inline packages/logger/src/index.spec.js:70]: same as above

**Review by stevengill — APPROVED** [review]

**Comment by stevengill** [comment]
Thanks Kaz!

**Comment by 43081j** [comment]
blast from the past :D thanks a lot for getting it all merged and caught up/reworked

**Pull request merged**